setup.py

```diff
@@ -40,6 +40,7 @@
         'pycodestyle',
         'pydocstyle',
         'pyflakes',
+        'rope',
```

Comment:
Please change this to

    rope>=0.10.5

which supports Python 2 and 3 using the same source base.